package.json

```diff
@@ -1,31 +1,122 @@
 {
-    "name": "noir-programming-language-syntax-highlighter",
-    "displayName": "Noir Programming Language Syntax Highlighter",
-    "description": "A syntax highlighter for the Noir Programming Language",
+    "name": "vscode-noir",
+    "displayName": "Noir",
+    "description": "Noir support for Visual Studio Code",
     "version": "0.0.1",
+    "publisher": "noir-lang",
+    "keywords": [
+        "noir",
+        "noir-lang"
+    ],
     "engines": {
-        "vscode": "^1.60.0"
+        "vscode": "^1.67.0"
```

Comment:
This is quite old still and we could upgrade even further

Reply:
soon :)